### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by shyusu4**
## PR Description

- Understood the concept of state management.
- Used store, actions, and reducers in React.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    case ADD_BOOK:
+      return [
+        ...state,
+        action.payload,
+      ];
+
+    case REMOVE_BOOK:
+      return [
+        state.map((book) => book.id !== action.payload),
+      ];
```
Comment (brytebee) [inline src/redux/books/books.js:16]: - [ ] **Required:** This function is required to return the list of Books without the one removed. Kindly implement a remove function.

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+const REMOVE_BOOK = 'bookstore-cms/books/REMOVE_BOOK';
+const initialState = [];
+
+export default function booksReducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        action.payload,
+      ];
```
Comment (brytebee) [inline src/redux/books/books.js:11]: - [ ] **Required:** This function is required to add to the list of Books. Kindly implement a add function.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @shyusu4👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- App functions as it should.✔️
- The `README` looks professional.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shyusu4** [comment]
Hi @brytebee, thanks for the review! I'll review my code and will try my best to implement the required changes. 😄

**Review by gemmen29 — APPROVED** [review]
Hi @shyusu4, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**